hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -489,8 +494,8 @@ module top_${top["name"]} #(
     .host_req_rdy_o  (flash_host_req_rdy),
     .host_req_done_o (flash_host_req_done),
     .host_rdata_o    (flash_host_rdata),
-    .flash_ctrl_i    (flash_req),
-    .flash_ctrl_o    (flash_rsp)
+    .flash_ctrl_i    (${m["inter_signal_list"][0]["top_signame"]}_req),
```

Comment:
btw, how do you think this should work if a module has multiple such inter-module connections?
Instead of the '0', i guess we would need to find the index related to each port name? Is that right?

Reply:
o i see, you have this below already. nevermind this comment.